File: Android/PersonitaSqlite/app/src/main/java/com/example/personitasqlite/DBHelper.java
package com.example.personitasqlite;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import androidx.annotation.Nullable;

public class DBHelper extends SQLiteOpenHelper {

    private static String DB_NAME = "base_personita.db";
    private static int DB_VERSION = 1;

    public DBHelper(@Nullable Context context) {
        super(context, DB_NAME, null, DB_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase base) {
        String comandoSql = "CREATE TABLE personita(nombre VARCHAR(50))";
        base.execSQL(comandoSql);

    }

    @Override
    public void onUpgrade(SQLiteDatabase base, int i, int i1) {
        String comandoSql = "DROP TABLE personita";
        base.execSQL(comandoSql);
        onCreate(base);
    }
}

File: Android/personaSQLite/app/src/main/java/com/example/personotasqlite/DBHelper.kt
package com.example.personotasqlite

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class DBHelper(context: Context?) :
    SQLiteOpenHelper(context, DB_NAME, null, DB_VERSION) {
    override fun onCreate(base: SQLiteDatabase) {
        val comandoSql =
            "CREATE TABLE personita(id INTEGER PRIMARY KEY AUTOINCREMENT, nombre VARCHAR(50), apPat VARCHAR(50), apMat VARCHAR(50), fechaNac TEXT, sexo VARCHAR(50))"
        base.execSQL(comandoSql)
    }

    override fun onUpgrade(base: SQLiteDatabase, i: Int, i1: Int) {
        val comandoSql = "DROP TABLE personita"
        base.execSQL(comandoSql)
        onCreate(base)
    }

    companion object {
        private const val DB_NAME = "personota.db"
        private const val DB_VERSION = 1
    }
}